file-share-service-desktop-client/SaveFileDialogService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+
+namespace UKHO.FileShareService.DesktopClient
+{
+    public interface ISaveFileDialogService
+    {
+        string SaveFileDialog(string fileName);
+    }
+    public class SaveFileDialogService : ISaveFileDialogService
+    {
+
+        public String SaveFileDialog(string fileName)
```

Comment:
A very minor change. Please change to lowercase string for consistency
```suggestion
        public string SaveFileDialog(string fileName)
```

Reply:
resolved